domain_scraper/scrape_domains.py

```diff
@@ -0,0 +1,221 @@
+"""Argument parser for domain-scraper module"""
+
+import email
+import json
+import logging
+import os
+import shutil
+import ssl
+import smtplib
+import re
+import sys
+from argparse import Namespace
+from collections import defaultdict
+from datetime import date
+from email.message import Message
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from time import time
+
+import pkg_resources
+from jinja2 import Template
+
+
+from .config import DEVELOPMENT as CONFIG
+from .parser import parse_arguments
+
+logger = logging.getLogger(__name__)
+
+
+class MessageDomains:
+    """Represents scraped domains from regular email Message object"""
+
+    REGEX_FROM = re.compile(r"from\s+(.+?)\s+(by|via|with|id|for|;)")
+    REGEX_BY = re.compile(r"by\s+(.+?)\s+(from|via|with|id|for|;)")
+
+    def __init__(self, msg: Message):
```

Comment:
Please add method description. Please check style guide what i shared with you

Reply:
Done